pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -725,17 +725,13 @@ func (a *Operator) copyToNamespace(csv *v1alpha1.ClusterServiceVersion, namespac
 			}
 		}
 
-		logger.Debug("checking status")
-		newCSV.Status = csv.Status
-		newCSV.Status.Reason = v1alpha1.CSVReasonCopied
-		newCSV.Status.Message = fmt.Sprintf("The operator is running in %s but is managing this namespace", csv.GetNamespace())
-
-		if !reflect.DeepEqual(fetchedCSV.Status, newCSV.Status) {
-			logger.Debug("updating status")
-			// Must use fetchedCSV because UpdateStatus(...) checks resource UID.
-			fetchedCSV.Status = newCSV.Status
-			if fetchedCSV, err = a.client.OperatorsV1alpha1().ClusterServiceVersions(namespace).UpdateStatus(context.TODO(), fetchedCSV, metav1.UpdateOptions{}); err != nil {
-				logger.WithError(err).Error("status update for target CSV failed")
+		logger.Debug("checking spec")
+		if !reflect.DeepEqual(fetchedCSV.Spec, newCSV.Spec) {
+			// update based on changes to the spec only
+			// Note: PUT verbs on objects (such as via Update()) ignore status values
+			fetchedCSV.Spec = newCSV.Spec
+			if fetchedCSV, err = a.client.OperatorsV1alpha1().ClusterServiceVersions(namespace).Update(context.TODO(), fetchedCSV, metav1.UpdateOptions{}); err != nil {
```

Comment:
It would be nice to wrap changes for both updated labels and spec into a single `Update` call.